R/rlistings_methods.R

```diff
@@ -200,18 +215,21 @@ setMethod(
 )
 
 #' @rdname listing_methods
+#' @param obj The object.
 #' @export
 setMethod(
   "subtitles", "listing_df",
   function(obj) attr(obj, "subtitles") %||% character()
 )
 #' @rdname listing_methods
+#' @param obj The object.
```

Comment:
This does not need to be repeated every time when al the methoeds are sharing an rd file